obsRecorder.js

```diff
@@ -1,22 +1,27 @@
 const path = require('path');
 const { Subject } = require('rxjs');
 const { first } = require('rxjs/operators');
+const { ipcMain } = require('electron');
```

Comment:
I can't find any usages of it. Is it needed?

Reply:
No, you're right. I forgot to delete that line.